ITEM.tsv

```diff
@@ -5742,18 +5742,18 @@ ITEM_20150717_005741	It was named as the appearance of a necklace. There are key
 ITEM_20150717_005742	It was named as the appearance of a necklace. There are keys of hidden places among these type necklaces. However, the place is not special, though the necklace is better special.
 ITEM_20150717_005743	There is a transmission that the first Master Sage crafted it.
 ITEM_20150717_005744	Long ago, when one magician's attempt to come back to the world as an Undead failed, only the magic remained in his bones. Other magicians used these bones to make the objects with magical power in them and this is one of their results
-ITEM_20150717_005745	Misfortune is said to follow this necklace but that is not true. Focus on the strenth it give its wearer. 
+ITEM_20150717_005745	Misfortune is said to follow this necklace but that is not true. Focus on the strength it give its wearer. 
 ITEM_20150717_005746	Even when Kepas were harvested as crops, before turning into monsters, They were known to be good for your strength. 
 ITEM_20150717_005747	It was originally made for Miners working in the dark underground but people of various jobs benefited from this necklace. It was first made in Klaipeda. 
 ITEM_20150717_005748	This necklace protected its owner until his last moment. We hope the new owner of this necklace always receives the protection from it.
-ITEM_20150717_005749	Since the Day of Divine Tree, The new skill obtained from new monsters has become known and new products have been manufactured based on the skill. And it is one among the products.
+ITEM_20150717_005749	Since the Medis Diena, The new skill obtained from new monsters has become known and new products have been manufactured based on the skill. And it is one among the products.
```

Comment:
Another Awkward sentence. Is it really 'skills'?
Maybe "..., new monsters brought new skills that were used to manufacture items such as this. It is a premier product.